### PR TITLE
fix: use relative base path for playground

### PR DESCRIPTION
**Pull request by lukashass**

### CODE REVIEW

File: packages/former/vite.config.playground.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { defineConfig } from 'vite';
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  base: '',
```
Comment (mariusheine) [inline packages/former/vite.config.playground.ts:7]: @lukashass No need to hardcode it in the vite config. Just run `pnpm run build:playground --base /former/` in the playground.yml insteadof just `pnpm run build:playground`.

**Comment by mariusheine** [comment]
@lukashass In order to test it, let's just deploy the playground also per PR with a base `former-pr-${PR_NUMBER}`. What do you think?

So some kind of review environment.

**Comment by lukashass** [comment]
> @lukashass In order to test it, let's just deploy the playground also per PR with a base `former-pr-${PR_NUMBER}`. What do you think?
> 
> So some kind of review environment.

@mariusheine I'm not sure if that is how github pages work. The subpath is the repo name AFAIK.

File: packages/former/vite.config.playground.ts (hunk 1 above)
Comment (lukashass) [reply packages/former/vite.config.playground.ts]: :+1:

**Comment by mariusheine** [comment]
> > @lukashass In order to test it, let's just deploy the playground also per PR with a base `former-pr-${PR_NUMBER}`. What do you think?
> > So some kind of review environment.
> 
> @mariusheine I'm not sure if that is how github pages work. The subpath is the repo name AFAIK.

@lukashass Ahh yeah you are right. We can't pick a page name by ourself

**Review by mariusheine — APPROVED** [review]

**Pull request merged**